src/lib/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -233,9 +227,34 @@ class PatronCurrentDocumentRequests extends Component {
     this.setState({ isSuccessMessageVisible: false, successMessage: '' });
   };
 
+  onShowAll = () => {
+    const { patronPid, fetchPatronDocumentRequests } = this.props;
+    fetchPatronDocumentRequests(patronPid);
+  };
+
+  onShowLess = () => {
+    this.fetchPatronDocumentRequests();
+  };
+
+  renderPagination = () => {
+    const { isLoading, documentRequests, rowsPerPage } = this.props;
+    const { activePage } = this.state;
+
+    return (
+      <RenderPagination
+        currentPage={activePage}
+        currentSize={rowsPerPage}
+        loading={isLoading}
+        onPageChange={this.onPageChange}
+        items={documentRequests}
+      />
+    );
+  };
+
   render() {
     const { documentRequests, isLoading, error, rowsPerPage } = this.props;
     const { activePage, isSuccessMessageVisible, successMessage } = this.state;
+    const content = `Your requests for new literature (${documentRequests.hits.length}/${documentRequests.total})`;
```

Comment:
I would keep only total number `(16)` as discussed on the chat

Reply:
I saw there was an error in the last push (the last push was only that change). As you can see in the screenshots the change was already implemented.